core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -80,11 +81,13 @@ class BaseColumnFormatRelation(
     with PartitionedDataSourceScan
     with RowInsertableRelation {
 
+  val scanAsUnsafeRows = false
```

Comment:
Until the code the stabilizes, we should have a SQLConf parameter instead of a global constant to disable the vectorized approach. This would help in TPCH testing to avoid restarting of the servers if we want to try performance of other type of scan. However, for now, the scan is always set as cached batch scan.